docs/development.md

```diff
@@ -134,6 +134,37 @@ in your boxes.yaml entry to configure this on box creation.
 #### Additional info
 - `foreman start` can be run with `rails` or `webpack` to start just that server: i.e. `bundle exec foreman start rails`. This can be used to start servers separately.
 
+### Customizing the Development Environment
+
+#### Custom local files
+
+You can have files automatically copied over to the `vagrant` user's home directory when spinning up a `centos7-katello-devel` or `centos7-katello-devel-stable` box. To do so, follow these steps:
```

Comment:
Technically this is not tied to vagrant and should work in any development setup right?